### ISSUE TITLE
TinyMCE should not encode special chars

### ISSUE THREAD

**Issue by sebastienros** [OrchardCMS/Orchard#5578]
Description of the issue:
https://medium.com/@urbanit/tinymce-orchardcms-f63b7271637c

The solution is to let the default setting, 'named', which means the chars are displayed as is, unless for specific known conversion. This patch defines all invisible and space entities.

More info and a patch are available on Drupal's forum: https://www.drupal.org/node/373542

**Comment by urbanit** [comment]
My experience indicates that "raw" is pretty much needed for non latin character languages. Otherwise code is unreadable.

**Comment by Jetski5822** [comment]
This should be added as a conditional statement to the TinyMCE module based on the Content Culture.

**Comment by Jetski5822** [comment]
NOTE: http://stackoverflow.com/questions/6127277/how-to-determine-if-a-cultureinfo-instance-supports-latin-characters

**Comment by sebastienros** [comment]
So, you actually all agree with the suggested solution ;) you just need to read the links content in full.

**Comment by Piedone** [comment]
@Jetski5822 I don't think this is related to the current culture. After all you could e.g. add an accented Hungarian quote to an otherwise English content.

**Comment by Jetski5822** [comment]
Yes I agree. 

but @Piedone - this is Europe, no one speaks Hungarian, everyone speaks English!. :)

**Comment by DanielStolt** [comment]
Second @Piedone - this should not be conditional based on culture.

Makes sense to me to keep the "named" setting and instead redefine the list of conversions to include a smaller set.

**Comment by barthamark** [comment]
Fixed in this changeset: https://github.com/OrchardCMS/Orchard/commit/188fabe233c3c9ebef04ccd29c1a07e8a520e882
